### PR TITLE
Fix link error

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: source/ntu/printer/index.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ For Windows, it is the same as the above manual, except the following parts:
 Using Printer at Ubuntu
 ------------------------
 
-1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`_ first if using wired connection
```
Comment (core-man) [inline source/ntu/printer/index.rst:65]:
```suggestion
1.  `Configure PEAP </ntu/spms-network>` first if using wired connection
```

File: source/ntu/printer/index.rst [hunk 2]
```diff
@@ -62,7 +62,7 @@ For Windows, it is the same as the above manual, except the following parts:
 Using Printer at Ubuntu
 ------------------------
 
-1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+1.  `Configure PEAP </ntu/spms-network>` first if using wired connection
```
Comment (core-man) [inline source/ntu/printer/index.rst:65]:
```suggestion
1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`_ first if using wired connection
```

File: source/ntu/printer/index.rst [hunk 3]
```diff
@@ -62,7 +62,7 @@ For Windows, it is the same as the above manual, except the following parts:
 Using Printer at Ubuntu
 ------------------------
 
-1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+1.  :doc:`Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`_ first if using wired connection
```
Comment (core-man) [inline source/ntu/printer/index.rst:65]:
```suggestion
1.  :doc:`Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>` first if using wired connection
```

File: source/ntu/printer/index.rst [hunk 4]
```diff
@@ -62,7 +62,7 @@ For Windows, it is the same as the above manual, except the following parts:
 Using Printer at Ubuntu
 ------------------------
 
-1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+1.  :doc:`Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>` first if using wired connection
```
Comment (core-man) [inline source/ntu/printer/index.rst:65]:
```suggestion
1.  :doc:`Configure PEAP </ntu/spms-network#access-intranet-via-wired-connection>` first if using wired connection
```

File: source/ntu/printer/index.rst [hunk 5]
```diff
@@ -62,7 +62,7 @@ For Windows, it is the same as the above manual, except the following parts:
 Using Printer at Ubuntu
 ------------------------
 
-1.  `Configure PEAP </ntu/spms-network/#access-intranet-via-wired-connection>`__ first if using wired connection
+1.  :ref:`Configure PEAP <intranet-wired-connection>` first if using wired connection
```
Comment (core-man) [inline source/ntu/printer/index.rst:65]: Fix the wrong reference link~

**Review by HouseJaay — APPROVED** [review]

**Pull request merged**